src/Accessibility/AXHearingUtilities.cs

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright 2021 Microsoft Corp
+//
+// Authors:
+//   Rachel Kang (rachelkang@microsoft.com)
+//
+
+#nullable enable
+
+#if !TVOS && !MONOMAC
+
+using System;
+using System.Runtime.InteropServices;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace Accessibility {
+    public static partial class AXHearingUtilities {
+
+        [iOS (15,0), NoMac, NoTV, Watch (8,0)]
```

Comment:
since the bindings already have this (partial) type decorated with the attributes then they are not needed on the members (since they are identical and are inherited from the generated type-level declaration).

To ease readability you can add 
```csharp
// accessibility.cs already provide the following attributes on the type
// [iOS (15,0), NoMac, NoTV, Watch (8,0)]
```
above line #19.

Reply:
Avoid marking comments as "resolved". It makes it hard for the reviewer to see if/how the comment was resolved.

It's better to ket the commenter mark it as "resolved" after it's been validated.

Reply:
sounds good, sorry about that!